src/compiler/crystal/codegen/fun.cr

```diff
@@ -95,7 +95,12 @@ class Crystal::CodeGenVisitor
         emit_def_debug_metadata target_def unless @debug.none?
         set_current_debug_location target_def if @debug.line_numbers?
 
+        {% if LibLLVM::IS_LT_150 %}
         context.fun.add_attribute LLVM::Attribute::UWTable
+        {% else %}
+          context.fun.add_attribute LLVM::Attribute::UWTable, value: LLVM::UWTableKind::Async
+        {% end %}
+
         if @program.has_flag?("darwin")
```

Comment:
Completely unrelated, but shouldn't this be a macro if?

Reply:
The whole `compare_versions(Crystal::LLVM_VERSION, "8.0.0") < 0` is no longer present on the master branch due to #12906